behavior/microservice/app.py

```diff
@@ -41,10 +42,122 @@ def infer(model_type, ou_type):
     return jsonify(Y)
 
 
+def connect():
+    # pylint: disable=assigning-non-slot
+    db = getattr(g, "__database", None)
+    if db is None:
+        db = g.__database = sqlite3.connect("inference.db")
+    return db
+
+
+@app.teardown_appcontext
+def close_connection(_):
+    db = getattr(g, "__database", None)
+    if db is not None:
+        db.close()
+
+
+def init_db():
```

Comment:
Since this isn't exposed as a route,
```suggestion
def _init_db():
```

?